packages/network-clients/src/clients/networkClient.ts

```diff
@@ -28,14 +28,21 @@ export class NetworkClient {
     this._contractClient = new ContractClient(_sdk);
   }
 
-  public static create(network: SQNetworks, provider?: Provider, ipfsUrl?: string) {
+  public static create(
+    network: SQNetworks,
+    provider?: Provider,
+    ipfsUrl?: string,
+    options?: {
+      queryClient?: GraphqlQueryClient;
```

Comment:
Problem still exists, that `GraphqlQueryClient` doesn't allow you override gql endpoint.